src/style-value.js

```diff
@@ -12,24 +12,23 @@
 //     See the License for the specific language governing permissions and
 // limitations under the License.
 
-(function(shared, testing) {
+(function(internal, scope, testing) {
 
-  function StyleValue() {
-  }
+  function StyleValue() {}
 
   // TODO: Include parsing logic here.
   StyleValue.parse = function(property, value) {
     if (typeof value == 'string') {
       numberValue = Number.parseFloat(value);
       if (numberValue !== NaN) {
-        return new NumberValue(numberValue);
+        return new scope.NumberValue(numberValue);
```

Comment:
We're doing scope.Thing everywhere else, and I think it should be consistent. But there is certainly room for discussion on the actual utility of having a scope that is window - but lets address it in a new PR.